tests/README.md

```diff
@@ -1,12 +1,38 @@
 # Oasis Indexer Tests
 
-This directory contains, primarily, e2e tests for the Oasis Indexer. They have been organized as follows:
+This directory contains tests for the Oasis Indexer. They have been organized as follows:
 
+- `e2e/` contains tests that run against a fake blockchain (provided by `oasis-net-runner`) and a black-box indexer (i.e. only external APIs are accessed).
 - `genesis/` contains tests that validate our database state is correct against `oasis-node` genesis state.
 - `http/` contains tests that validate API endpoints behave as expected and return correct output.
 
-## Setup for E2E Tests
+## E2E Tests
 
-To ensure that tests behave as expected, you should have either the local environment or Docker environment configured as per the [top-level README](../README.md#docker-development).
+To ensure that tests behave as expected, you should have Docker environment configured as per the [top-level README](../README.md#docker-development).
 
-Then you can run these tests from the repository root with `make test-ci` or directly from this directory with `go test ./... -v`.
+### Running e2e tests locally
+
+For the canonical way to run these tests, see the [CI config](../../.github/workflows/ci-test.yaml).
+Here are some useful one-liners derived from those commands, and adapted for running locally and repeatedly.
+
+### Get docker containers with current code up&running
+```sh
+make docker &&
+{ docker compose -f tests/e2e/docker-compose.e2e.yml down -t 0 || true; } &&
+make clean-e2e &&
+env HOST_UID=$(id -u) HOST_GID=$(id -g) docker compose -f tests/e2e/docker-compose.e2e.yml up -d
```

Comment:
Optional: Now that you've taken on creating higher-level makefile recipes, I would suggest renaming the old `test-e2e` to `test-e2e-local`, and creating a new `test-e2e` that consists of this line and line 32.

Then the steps for running e2e become pretty digestible: `make docker && make clean-e2e && make test-e2e`

Reply:
Used `start-e2e` to describe what you intend by `test-e2e` here I think.

Reply:
I had a hard time explaining ... I pushed a commit with what I had in mind. Your call on whether you want to include it or go with what you prepared. I'm OK with both; the reason I prefer my edit is that the commands for running tests locally are then concise. (That's the commands I care about; it shouldn't take massive history/docs hunting and long copy-paste to run tests. In CI, everything is in code and hands-off so it's less of a problem if commands are a little longer there.)